README.md

```diff
@@ -55,3 +55,7 @@ The following screenshots illustrate the preconfigured dashboards in this reposi
 #### Prometheus Statistics
 ![Prometheus Statistics](doc/DSEMetricsCollectorDashboardPrometheus.png)
 
+## Kubernetes Support
+These dashboards and prometheus configuration files may be transformed via the collection of Python scripts under `bin/`.
+
+Specifically run `bin/clean.py && bin/build.py` to generate appropriate files in the `dist` directory. These files integrate with the Custom Resources defined by the [Prometheus](https://operatorhub.io/operator/prometheus) and [Grafana](https://operatorhub.io/operator/grafana-operator) operators available on Operator Hub.
```

Comment:
An idea: rather than packaging up the dashboard json into the yamls could we just point to them using this feature?
https://github.com/integr8ly/grafana-operator/commit/0cceed5906477ebb03ddc77b4eba36be37add63c

Reply:
example here https://github.com/integr8ly/grafana-operator/blob/0cceed5906477ebb03ddc77b4eba36be37add63c/deploy/examples/dashboards/DashboardFromURL.yaml#L9

Reply:
Yep, that's an awesome feature. I wonder how often it pulls from that URL. If it's just on resource creation or if there is a chance it gets changed when we push an update to git (if we point to a branch-based URL, not an individual commit).

Reply:
> *url*: Url address to download a json string with the dashboard contents. This will take priority over the json field in case the download is successful

This makes me think we would want both. What are your thoughts @phact ?

Reply:
> Yep, that's an awesome feature. I wonder how often it pulls from that URL. If it's just on resource creation or if there is a chance it gets changed when we push an update to git (if we point to a branch-based URL, not an individual commit).

My guess is on resource creation or upgrade but it's worth looking into to confirm. Good point on branch vs commit distinction, this would be painful to update on every commit.

> > _url_: Url address to download a json string with the dashboard contents. This will take priority over the json field in case the download is successful
> 
> This makes me think we would want both. What are your thoughts @phact ?

Yeah, if github's not available or someone's deploying inside a firewall it might be nice to have a fallback. At the same time these feel like minor edge cases.

The reason I looked into the url option is we're talking about a lot of json and it feels kudgey to inline it into a yaml like that. Also we'd have to add automation because folks are bound to forget to run the scrips on commit. Is this how folks are solving this kind of problem in k8s?

Another way to do it would be to rely on the existing docker containers which include the dashboard jsons as part of the docker build. Though I don't know if there's a way to have this play nice with the integreatly stuff you're using.

Reply:
We should probably automate some of the dashboards with jsonnet anyway, then we'd have a build and could just add this as an extension point to that.